03LessonThree/in-class-project/daily-planet.html

```diff
@@ -1,10 +1,35 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <title>The Daily Planet</title>
-        <link rel="stylesheet" href="css/daily-planet-style.css"/>
-    </head>
-    <body>
-        <!-- You code start here -->
-    </body>
-</html>
\ No newline at end of file
+
+<head>
+    <title>The Daily Planet</title>
+    <link rel="stylesheet" href="css/daily-planet-style.css" />
+</head>
+
+<body>
+    <header>
+        <main>
+          <div></div>
+            <h1>The Daily Planet</h1>
+            <nav><a href="http://espn.com">Sports</a>
+                <a href="http://do512.com">Local</a>
+                <a href="http://dowjones.com">Economy</a>
+            </nav>
+            <section>
+                <article><h3>Giant Hello Kitty Attacks!</h3>
+                  <h5><em>Written by: Lois Lane</em></h5>
+                <img src="https://s-media-cache-ak0.pinimg.com/564x/5e/9c/90/5e9c904179d02da70e80ea252219d0fe.jpg"></article>
+                <p>In an amazing show of manliness, Superman has once again saved our city! Early yesterday morning, a large
+                   white giant kitten (aka "Hello Kitty") waddled down our Main Street.</p>
+                   <p>Superman, hearing the invasion with his super hearing, spead to the scene of
+                     litter box and lured the kitty away with a lure of catnip.</p>
+            </section>
+            <aside><img src="https://m1.behance.net/rendition/modules/62607987/disp/396cba801481fd221fe0324f918cf1c6.jpg"></a>
+            </aside>
+        </main>
+    </header>
+    <footer></footer>

```

Comment:
You are missing this project requirement: Add copyright of the company and the current year.

Add it in and commit again, and I can bump your grade up higher.